packages/ui/src/utils/utils.ts

```diff
@@ -70,14 +70,36 @@ export const timeDistance = (date1 = 0, date2 = 0, onlyHighestDenomination = fal
  * @example
  * timeAgo(1708113838167) // Returns "1 hour ago"
  */
+
 export const timeAgo = (timestamp?: number | null): string => {
   if (timestamp === null || timestamp === undefined) {
     return 'Unknown time'
   }
 
+  const now = Date.now()
+  const threeDaysMs = 3 * 24 * 60 * 60 * 1000
+  const isOld = now - timestamp > threeDaysMs
+
+  if (isOld) {
+    const date = new Date(timestamp)
+    return (
+      new Intl.DateTimeFormat(LOCALE, {
+        month: 'short',
+        day: 'numeric',
+        year: 'numeric'
+      }).format(date) +
+      ' at ' +
```

Comment:
probably a product decision, but I don't mind if we remove the `at` and just use one `Intl()` to show the complete date + time string